chart/templates/db-initscripts-configmap.yaml

```diff
@@ -17,7 +16,6 @@ metadata:
 data:
   init.sql: |-
 {{- $root := . }}
-{{- range $path, $bytes := .Files.Glob "config/db/init/**.sql" }}
+{{- range $path, $bytes := .Files.Glob "config/db/init/02-create-and-init-sessions-db.sql" }}
```

Comment:
Could you elaborate on this change? Why not are the other scripts not needed anymore?

Reply:
@geropl because `03-recreate-gitpod-db` drops the db.

Reply:
> Does it seem the scripts expect the chart will be used for testing?

They are used for both use cases: initialization (because it's straight-forward to mount into any DB container) and testing.

> If you provide a connection to MySQL we should only assume the user, password, and database are valid. Everything else should be done by typeorm (even the creation of the sessions db)

That's certainly another way to do it. What would be the advantage over the current approach?